multi-device + lowdb

BochilGaming opened the pull request:
**Siapa yang disini sering males ngebot karena internet ngelag adik-adik?** :v

- [x] Working
- [x] Fix session error ketika restart
- [x] Fix getName
- [x] Fix serialized message
- [x] Fix print message
- [ ] Fix jadibot
- [ ] Fix bug
- [ ] #156 
- [x] #83
- [ ] #8 
- [ ] #111 
- [ ] #38, #73
- [x] #389, #32 
- [x] ESM
- [x] Support multiple platform?
- [x] Nurutomo/wabot-aq#490
- [x] #248 
- [x] #262 
- [x] Reduce global object

BochilGaming committed:
initial

Dawnfrosty commented:
> > Ini bisa langsung dipake?
> 
> udh tapi blm bisa apa"

Bang ada official grup games-wabot ga?

Dawnfrosty commented:
> > > > Ini bisa langsung dipake?
> > > 
> > > 
> > > udh tapi blm bisa apa"
> > 
> > 
> > Bang ada official grup games-wabot ga?
> 
> ada keknya

Ada link nya?

zatu22 commented:
Dont forget, every new session must remain open for at least 5 minutes in order to be saved successfully

BochilGaming commented:
> Dont forget, every new session must remain open for at least 5 minutes in order to be saved successfully

it depends on how many messages you have, and your internet bot

BochilGaming committed:
Fixed some bugs

BochilGaming committed:
Merge branch 'main' into multi-device

BochilGaming committed:
Fix: ?

BochilGaming committed:
fix some bug

BochilGaming committed:
fix in nodejs <12v

bytesWhisperer committed:
MultiDevice few bug fix (#146)

* Bug fix

Bug fix

* .....

Co-authored-by: klstefan <74061098+klstefan@users.noreply.github.com>
Co-authored-by: BochilGaming <79433517+BochilGaming@users.noreply.github.com>

bytesWhisperer committed:
[Multi device] Fix (#158)

* Bug fix

-Added missing node modules & fix
-Fixed print message
-Fixed global.owner error
- New entry gitignore
=MultiDevice is working now

* Update simple.js

* Update handler.js

BochilGaming committed:
fix spam in console

https://www.bleepingcomputer.com/news/security/dev-corrupts-npm-libs-colors-and-faker-breaking-thousands-of-apps/

ZeroChanBot committed:
fitur mancing (#62)

BochilGaming committed:
move to esm

BochilGaming committed:
Update config.yml

BochilGaming committed:
Updated config.yml

BochilGaming committed:
Updated config.yml

BochilGaming committed:
Updated config.yml

BochilGaming committed:
Updated config.yml

BochilGaming committed:
Fix? #220

Maybe this fix issue #220

BochilGaming committed:
Fix :v

DineshValor committed:
fix user pp fetch (#225)

DineshValor committed:
Update youtube-play.js (#229)

BochilGaming committed:
Maybe fix? #227

BochilGaming committed:
updates
add delete.js, close #228
maybe fix? #233
implement Nurutomo/wabot-aq#490 (beta)
fix hidetag!
and mores...

jauhariel committed:
db? (#238)

DineshValor committed:
Update README.md (#236)

DineshValor committed:
Merge with tools menu (#251)

Created sperate menu with name tool, but already exist Tools menu, so merging them.

BochilGaming committed:
Updates and fix bugs
1.Bug creator, fix #248
2.Fix /okick #247
3.Add antilink #247
4.new features

BochilGaming committed:
Updated config.yml

BochilGaming committed:
Updated config.yml

BochilGaming committed:
Updated config.yml

BochilGaming committed:
Updated config.yml

BochilGaming committed:
Updated config.yml

BochilGaming committed:
Updated config.yml

BochilGaming committed:
Updated config.yml

BochilGaming committed:
Updated config.yml

BochilGaming committed:
install qrcode-terminal

Maybe fix ? #254

BochilGaming committed:
Updated config.yml

BochilGaming committed:
Updated config.yml

BochilGaming committed:
Updated config.yml

DineshValor committed:
Fix 'no qr-terminal dependency' (#257)

For termux users.

DineshValor committed:
Update "steps for termux user" (#258)

* Update "steps for termux user"

Working Installation steps for termux users.

* Update README.md

Co-authored-by: BochilGaming <79433517+BochilGaming@users.noreply.github.com>

BochilGaming committed:
Fix m.getQuotedObj

Commit:
dare button (#266)

Commit:
truth button (#265)

Commit:
update button (#264)

Co-authored-by: BochilGaming <79433517+BochilGaming@users.noreply.github.com>

DineshValor committed:
Update gitclone.js (#267)

DineshValor committed:
Error steps for termux user (#260)

Added steps, if error in "yarn / node ." for Termux User.

BochilGaming committed:
Updates
1. Fix quoted in serialize message
2. Fixes #262, fixes #274, fixes #276
3. Throw error if sendFile with filesize >= 100 MB
4. New Features
5. Etc....

DineshValor committed:
Added `--autocleartmp` (#283)

Added `--autocleartmp` in readme.md

DineshValor committed:
Little changes (#287)

Improve english words as simpler form.

aniq12 committed:
Add cmdWithMedia & Tagall (#290)

* Create group-tagall.js

* Create cmd-del.js

* Create cmd-set

* Create cmd-list.js

* Rename cmd-set to cmd-set.js

* Create _cmdWithMedia.js

Close #291

BochilGaming commented:
> Ini otomatis ngehapus dalam waktu tertentu? Atau dalam ukuran tertentu ya?

Iy, tapi ukuran tidak mempengaruhi

BochilGaming committed:
Bump @bochilteam/scraper

Commit:
Create downloader-mediafirejs (#301)

* Create downloader-mediafirejs

* Update and rename downloader-mediafirejs to downloader-mediafire.js

Co-authored-by: BochilGaming <79433517+BochilGaming@users.noreply.github.com>

Commit:
Update quotes-bucin.js (#304)

* Update quotes-bucin.js

* Update quotes-bucin.js

aniq12 committed:
Leave Gc (#314)

* Create group-leavegc.js

* Fix leavegcall

Co-authored-by: BochilGaming <79433517+BochilGaming@users.noreply.github.com>

DineshValor committed:
Move to baileys stable version (#293)

* Update stable baileys version

Unstable baileys cause Termux user to show "@adiwajshing/baileys/ modules not found".

Keep version "<blank>" or use latest "^4.0.1"

* Update package.json

Maybe fix:
- https://github.com/BochilGaming/games-wabot/issues/317#issuecomment-1103561130

Co-authored-by: BochilGaming <79433517+BochilGaming@users.noreply.github.com>

BochilGaming committed:
Updates
1. Fix #309
2. Update save auth system

aniq12 committed:
fix cmdWithMedia (#324)

Fixes #323

aniq12 committed:
fix readviewonce (#326)

jauhariel committed:
Kunci command stiker (#333)

BochilGaming committed:
Reduce global variable
1. Reduce global variable to avoid memory leak (maybe) #331
2. Use MultiAuthState in session, it will make folder name sessions
3. db and conn no longer in global object

Commit:
Update cmd-set.js (#335)

BochilGaming committed:
Fix print 'cannot read property'

BochilGaming committed:
Fix autoread

BochilGaming committed:
Fix loadMessage if message undefined

BochilGaming committed:
Fix error: conn is not defined

BochilGaming committed:
Fix welcome/bye, and update owner-exec

jauhariel committed:
Banned user & unbanned user (#347)

* banned user

* unbanned user

BochilGaming committed:
Fix session in windows

BochilGaming committed:
Fix fetch

jauhariel committed:
Typo + kurang dikit (#355)

* Typo

* Update owner-unbanuser.js

* Update owner-banuser.js

aniq12 committed:
addmsg, delmsg, listmsg (#348)

* Create _getmsg.js

* Create msg-add.js

* Create msg-del.js

* Create msg-list.js

* Update msg-del.js

Co-authored-by: BochilGaming <79433517+BochilGaming@users.noreply.github.com>

BochilGaming committed:
Fix message delete

Fix #360

Commit:
update (#376)

BochilGaming committed:
Some bug fixes

BochilGaming committed:
.

BochilGaming committed:
Fix queque

BochilGaming committed:
Close after open

BochilGaming committed:
Fix some bugs
1. Fix missing message. Closes #384
2. Add role. Closes #389 Closes #32
3. Fix anti delete
4. Fix msg. Closes #360
5. Etc..

BochilGaming committed:
Delete TODO

BochilGaming committed:
Fix bug and new feature
1. Fix bug sendContact if number is bussines. Closes #390
2. Add how to convert single file to multi file

BochilGaming committed:
Merge branch 'multi-device' of https://github.com/BochilGaming/games-wabot into multi-device

raseldev18 committed:
anu (#392)

BochilGaming committed:
Fix mongodbv2

BochilGaming committed:
resolve

BochilGaming committed:
better resolve

Commit:
anu conn (#398)

Ftwrr committed:
Update anonymous_chat.js (#399)

BochilGaming committed:
Add support multi directory plugin and change buffer to stream

BochilGaming committed:
Merge branch 'multi-device' of https://github.com/BochilGaming/games-wabot into multi-device

BochilGaming committed:
Fix loading plugins in windows

BochilGaming committed:
delete debugging

BochilGaming committed:
Fix downloadM()

Co-authored-by: Rasel <raselcomel18@gmail.com>

BochilGaming committed:
Fixed a bug
